docs/guide/docs/tutorials/azure_ad_oauth2.md

```diff
@@ -0,0 +1,69 @@
+---
+id: azure-oauth2
+title: Setting up SSO with Azure AD OAuth2
+---
+
+**Note**: This is a Botpress Pro feature
+
+## Step 1: Create a tenant
+
+Navigate to Azure Active Directory Page in your Azure Portal and create a new tenant
+
+## Step 2: Register an application
+
+In the page of your newly created tenant navigate to "App registrations".
+
+![App Registrations](assets/oauth/az_2_register_app.png)
+
+Create a new application and choose the appropriate configuration for your use case in "Supported account types" and keep the "Redirect URI" blank for now.
+
+![App Creation](assets/oauth/az_2.1_app_creation.png)
+
+## Step 3: Configure Botpress
+
+In your Botpress instance navigate to the code editor by going into any of your bots and add a new entry within "authStrategies" (should be around line 229) in the _botpress.config.json_ file, you may name the strategy whatever you want (keep the name URL safe) and fill in the entry in the following way:
```

Comment:
```suggestion
In your Botpress instance navigate to the code editor by going into any of your bots and add a new entry within "authStrategies" in the _botpress.config.json_ file, you may name the strategy whatever you want (keep the name URL safe) and fill in the entry in the following way:
```

Reply:
(The line number is bound to change with time, and it's 100% certain we'll forget to update the docs accordingly)